src/main/kotlin/no/nav/syfo/client/arbeidsforhold/ArbeidsforholdClient.kt

```diff
@@ -15,12 +16,11 @@ import org.slf4j.LoggerFactory
 class ArbeidsforholdClient(
     private val azureAdClient: AzureAdClient,
     private val clientEnvironment: ClientEnvironment,
+    private val httpClient: HttpClient = httpClientDefault(),
```

Comment:
Bare nysgjerrig egentlig – må alle disse klientene våre ha en instanser av `HttpClient` eller kunne vi hatt en felles? 🤔

Reply:
Godt spørsmål. Vil tro vi kunne hatt en felles (eller en `httpClientDefault` og en av `httpClientProxy`).

Reply:
Ikke viktig å endre til det, lurte bare 😊 Grunnen til at det ligger som parameter er kanskje pga. tester?

Reply:
Ja, det er for testbarheten :+1: